### ISSUE TITLE
LHR problem

### ISSUE THREAD

**Issue by t0mb3r** [trexminer/T-Rex#688]
Hi Im running 4 3080ti with the same LHR-tune setting, and similar oc to each. Two of them get 86Mh while two of them get 72. Whats wrong?

**Comment by t0mb3r** [comment]
Yes I'm aware of the silicon lottery and about the difference in the hashrate between two identical cards. But what I mean is that when the miner starts, it shows two different target mh per card. Two say about 85 and two say about 75, and then they output about that performance. Shouldn't they target all 85 and then output whatever they really support? For me, it seems like a bug, or that something is out of wack. Every gpu has its own oc slightly different but similar, and all temps are in safe ranges. Core temp about 50-60 and mem about 90 for every card.

**Comment by bayesta** [comment]
> Hola, estoy ejecutando 4 3080ti con el mismo ajuste de sintonización LHR y oc similar para cada uno. Dos de ellos obtienen 86Mh mientras que dos obtienen 72. ¿Qué pasa?

a mi me alcanza los 80mh, el problema es con la temperatura de las memorias que me oscila entre 90 y 104 grados. que temperatura tienes tu en las memorias? es que no se si realizar el cambio de thermal pads
